File: trunk/C#/PokerProtocol/Commands/Game/GameEndedCommand.cs
using System;
using System.Collections.Generic;
using System.Text;
using EricUtility;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Game
{
    public class GameEndedCommand : AbstractCommand
    {
        protected override string CommandName
        {
            get { return COMMAND_NAME; }
        }
        public static string COMMAND_NAME = "gameENDED";


        public GameEndedCommand(StringTokenizer argsToken)
        {
        }

        public GameEndedCommand()
        {
        }
    }
}

File: JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/game/GameEndedCommand.java
package bluffinmuffin.protocol.commands.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands.AbstractCommand;

public class GameEndedCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return GameEndedCommand.COMMAND_NAME;
    }
    
    public static String COMMAND_NAME = "gameENDED";
    
    public GameEndedCommand(StringTokenizer argsToken)
    {
    }
    
    public GameEndedCommand()
    {
    }
}
